lib/grape/middleware/throttle_middleware.rb

```diff
@@ -11,6 +11,9 @@ def before
           period = 1.day
         elsif limit = throttle_options[:monthly]
           period = 1.month
+        elsif throttle_options[:period] && throttle_options[:limit]
```

Comment:
@jannsk Yeah, specifically the parameters to the middleware. For example, if no parameters at all are sent, raise an error to indicate that at minimum, a preset period must be set. If a period is set without a limit, we could perhaps default to a reasonable number (1, 3?), or raise an error. As for the format of the error, whatever you think is best. Other than just the usual subclass of `StandardError` I have no preference.

Sorry for the late reply, I was on vacation =)